pkg/process/config/yaml_config.go

```diff
@@ -153,6 +153,17 @@ func (a *AgentConfig) LoadProcessYamlConfig(path string) error {
 		}
 	}
 
+	// The maximum number of processes belonging to a container per message. Note: Only change if the defaults are causing issues.
+	if k := key(ns, "max_ctr_procs_per_message"); config.Datadog.IsSet(k) {
+		if maxCtrProcessesPerMessage := config.Datadog.GetInt(k); maxCtrProcessesPerMessage <= 0 {
+			log.Warn("Invalid max container processes count per message (<= 0), ignoring...")
```

Comment:
nit: instead of saying `ignoring` it would be more useful to say `use default value of 10000` or something more explicit.